csrc/build.rs

```diff
@@ -91,19 +94,10 @@ fn main() -> Result<()> {
 
     let out_dir = PathBuf::from(std::env::var("OUT_DIR").context("OUT_DIR not set")?);
     let build_dir = match std::env::var("ATOMA_FLASH_ATTN_BUILD_DIR") {
```

Comment:
will we ever use the custom build dir?

Reply:
I think that will be the case, if you want to save the cuda compiled binaries without having to compromise your `rust` build.